docs/graphQl/GettingStarted.md

```diff
@@ -0,0 +1,37 @@
+# GraphQl endpoint for FHIR-Server
+> A GraphQL prototype for the [HL7 FHIR specification](https://www.hl7.org/fhir/) based on the current implementation guide 
+> for [GraphQL with FHIR](https://build.fhir.org/graphql.html).
+ 
+## What Can I do with GraphQL in FHIR-Server?
+* ✅ Get all Patients in your FHIR-Server with specified properties based on [Patient Resource](https://www.hl7.org/fhir/patient.html).

```

Comment:
The endpoint is currently limited to 10 patients since there isn't a way to process continuation tokens. I think it would be good to call this out in the documentation so that users are aware.

Reply:
Thank you for pointing out, I changed to: **Get first 10 Patients in your FHIR-Server with specified properties based on Patient Resource.** and also, I added that to the **Next Steps and Important Notes**